### ISSUE TITLE
Cannot read property 'Inside' of undefined

### ISSUE THREAD

**Issue by fmsthird** [palerdot/react-d3-speedometer#81]
**react-d3-speedometer version**: 0.12.0
**react version**: 16.13.1
**typescript**: 3.7.2

**Question**

I was trying to implement _react-d3-speedometer_ using typescript.   
When I tried to apply position in _**customSegmentLabels**_ ,  I get an error `Type '"OUTSIDE"' is not assignable to type 'CustomSegmentLabelPosition'`. So I added it to imports but when I assigned it to position like this:  
`position: CustomSegmentLabelPosition.Inside,` then the error **Cannot read property 'Inside' of undefined**.

**Sample Codesandbox**  
[Sample](https://codesandbox.io/s/fervent-meitner-e4q9v?file=/src/App.tsx)

**Comment by palerdot** [comment]
@fmsthird 

Can your try casting the value? like
```
          {
            text: "Very Good",
            position: "INSIDE" as CustomSegmentLabelPosition,
            color: "#555"
          }
```
I agree this should work the way you tried. But for some reason, string enum is not available during run time. I will dig into this when I have time and update here. 

Meanwhile, please try casting the string `("INSIDE" as CustomSegmentLabelPosition,)` to unblock yourself for now.

[Sample sandbox for casting reference](https://codesandbox.io/s/focused-browser-fp2g6?file=/src/App.tsx:1070-1202)

**Comment by fmsthird** [comment]
Hi @palerdot 

That works! Thank you 👍 
Also, it would also be nice in the next update if the text will curve alongside with the bar. 🙂

**Comment by palerdot** [comment]
Hi,

I'm reopening this issue to keep track of this behavior. The right behavior is to do `CustomSegmentLabelPosition.Inside / CustomSegmentLabelPosition.Outside` ...

**Comment by palerdot** [comment]
Also regarding 
> Also, it would be nice in the next update if the text will curve alongside with the bar.
I'm tracking this here - https://github.com/palerdot/react-d3-speedometer/issues/82

**Comment by palerdot** [comment]
This issue is fixed in `v0.13.0`

There is proper Typescript enum support for `CustomLabelSegmentPosition` and `Transition`. This helper type/Object works both for Typescript and JS.

[Typescript Sandbox Example](https://codesandbox.io/s/bold-cloud-4os8p?file=/src/App.tsx)
[JS Sandbox Example](https://codesandbox.io/s/awesome-water-rbcm1?file=/src/App.js)

**Comment by Mmaheshbabu123** [comment]
./node_modules/we/lib/helpers.js:6:0
Module not found: Can't resolve 'fs'

Import trace for requested module:
./node_modules/we/bin/we
./pages/SpedoMeter.jsx

https://nextjs.org/docs/messages/module-not-found
![Screenshot from 2023-06-05 18-49-40](https://github.com/palerdot/react-d3-speedometer/assets/29708637/95dcc530-e25a-4e63-b79d-3434c1e819f8)
